tests/braille-specs/th_g0.yaml

```diff
@@ -0,0 +1,105 @@
+# encoding: utf-8 
+display: unicode.dis
+table:
+  language: th
+  region: th-TH
+  system: thai
+  __assert-match: th-g0.utb
+
+# forward test
+flags: {testmode: forward}
+tests:
+ - [ก ไก่, ⠛⠀⠱⠛⠔]
+ - [ข ไข่, ⠅⠀⠱⠅⠔]
+ - [ฃ ฃวด, ⠴⠅⠀⠴⠅⠺⠙]
+ - [ค ควาย, ⠥⠀⠥⠺⠡⠽]
+ - [ฅ ฅน, ⠤⠥⠀⠤⠥⠝]
+ - [ฆ ระฆัง, ⠠⠥⠀⠗⠁⠠⠥⠜⠻]
+ - [ง งู, ⠻⠀⠻⠒]
+ - [จ จาน, ⠚⠀⠚⠡⠝]
+ - [ฉ ฉิ่ง, ⠌⠀⠌⠃⠔⠻]
+ - [ช ช้าง, ⠬⠀⠬⠲⠡⠻]
+ - [ซ โซ่, ⠮⠀⠊⠮⠔]
+ - [ฌ กเฌอ, ⠠⠬⠀⠛⠋⠠⠬⠕]
+ - [ญ หญิง, ⠠⠽⠀⠓⠠⠽⠃⠻]
+ - [ฎ ชฎา, ⠠⠙⠀⠬⠠⠙⠡]
+ - [ฏ ปะฏัก, ⠠⠳⠀⠯⠁⠠⠳⠜⠛]
+ - [ฐ ฐาน, ⠠⠞⠀⠠⠞⠡⠝]
+ - [ฑ มนโฑ, ⠠⠾⠀⠍⠝⠊⠠⠾]
+ - [ฒ ผู้เฒ่า, ⠤⠾⠀⠏⠒⠲⠋⠤⠾⠔⠡]
+ - [ณ เณร, ⠠⠝⠀⠋⠠⠝⠗]
+ - [ด เด็ก, ⠙⠀⠋⠙⠄⠛]
+ - [ต เต่า, ⠳⠀⠋⠳⠔⠡]
+ - [ถ ถุง, ⠞⠀⠞⠉⠻]
+ - [ท ทหาร, ⠾⠀⠾⠓⠡⠗]
+ - [ธ ธง, ⠴⠾⠀⠴⠾⠻]
+ - [น หนู, ⠝⠀⠓⠝⠒]
+ - [บ ใบไม้, ⠧⠀⠱⠂⠧⠱⠍⠲]
+ - [ป ปลา, ⠯⠀⠯⠇⠡]
+ - [ผ ผึ้ง, ⠏⠀⠏⠪⠲⠻]
+ - [ฝ ฝา, ⠭⠀⠭⠡]
+ - [พ พาน, ⠹⠀⠹⠡⠝]
+ - [ฟ ฟัน, ⠫⠀⠫⠜⠝]
+ - [ภ สำเภา, ⠠⠹⠀⠎⠵⠋⠠⠹⠡]
+ - [ม ม้า, ⠍⠀⠍⠲⠡]
+ - [ย ยักษ์, ⠽⠀⠽⠜⠛⠤⠎⠴]
+ - [ร เรือ, ⠗⠀⠋⠗⠢⠕]
+ - [ล ลิง, ⠇⠀⠇⠃⠻]
+ - [ฤ พฤหัสบดี, ⠗⠂⠀⠹⠗⠂⠓⠜⠎⠧⠙⠆]
+ - [ว  แหวน, ⠺⠀⠀⠣⠓⠺⠝]
+ - [ศ ศาลา, ⠠⠎⠀⠠⠎⠡⠇⠡]
+ - [ษ  เมษา, ⠤⠎⠀⠀⠋⠍⠤⠎⠡]
+ - [ส เสือ, ⠎⠀⠋⠎⠢⠕]
+ - [ห หีบ, ⠓⠀⠓⠆⠧]
+ - [ฬ จุฬา, ⠠⠇⠀⠚⠉⠠⠇⠡]
+ - [ฦ, ⠇⠂]
+ - [อ อ่าง, ⠕⠀⠕⠔⠡⠻]
+ - [ฮ นกฮูก, ⠿⠀⠝⠛⠿⠒⠛]
+ - [ดินสอ ปากกา ฯลฯ, ⠙⠃⠝⠎⠕⠀⠯⠡⠛⠛⠡⠀⠰⠇]
+ - [มากมาย .......... จำไม่ได้, ⠍⠡⠛⠍⠡⠽⠀⠠⠠⠠⠠⠠⠠⠠⠠⠠⠠⠀⠚⠵⠱⠍⠔⠱⠙⠲]
+ - [กา ก่า ก้า ก๊า ก๋า, ⠛⠡⠀⠛⠔⠡⠀⠛⠲⠡⠀⠛⠶⠡⠀⠛⠦⠡]
+ - [เตะ, ⠋⠳⠁]
+ - [ด้วย, ⠙⠲⠺⠽]
+ - [ส่ง, ⠎⠔⠻]
+ - [จาก, ⠚⠡⠛]
+ - [แมน, ⠣⠍⠝]
+ - [กิน, ⠛⠃⠝]
+ - [กลืน, ⠛⠇⠢⠝]
+ - [ใบ, ⠱⠂⠧]
+ - [ต่าง ๆ, ⠳⠔⠡⠻⠀⠂]
+ - [ไป, ⠱⠯]
+ - [จำ, ⠚⠵]
+ - [จะ, ⠚⠁]
+ - [มัน, ⠍⠜⠝]
+ - [ผี, ⠏⠆]
+ - [ฤๅ ษี, ⠗⠂⠐⠡⠀⠤⠎⠆]
+ - [95/8, ⠼⠊⠑⠸⠌⠼⠓]
+ - [ลุง, ⠇⠉⠻]
+ - [แตงโม, ⠣⠳⠻⠊⠍]
+ - [เป็น, ⠋⠯⠄⠝]
+ - [จ๋า, ⠚⠦⠡]
+ - [ชม., ⠬⠍⠲]
+ - [1.5, ⠼⠁⠲⠑]
+ - [ธฺ ธรรมะ, ⠴⠾⠄⠀⠴⠾⠗⠗⠍⠁]
+ - [วันจันทร์, ⠺⠜⠝⠚⠜⠝⠾⠗⠴]
+ - [ไปใหน, ⠱⠯⠱⠂⠓⠝]
+ - [แดงว่า "เธอชื่ออะไร", ⠣⠙⠻⠺⠔⠡⠀⠦⠋⠴⠾⠕⠬⠢⠔⠕⠕⠁⠱⠗⠴]
+ - [สวดมนต์ ธรรมมํ ธรรมมัง, ⠎⠺⠙⠍⠝⠳⠴⠀⠴⠾⠗⠗⠍⠍⠐⠀⠴⠾⠗⠗⠍⠍⠜⠻]
+ - [ก๊อก, ⠛⠶⠕⠛]
+ - [กรุงเทพ ฯ, ⠛⠗⠉⠻⠋⠾⠹⠀⠰⠆]
+ - [๑+๑ เป็น ๒, ⠠⠼⠁⠐⠖⠠⠼⠁⠀⠋⠯⠄⠝⠀⠠⠼⠃]
+ - [รู้, ⠗⠒⠲]
+ - [ผมมี๑๐๐ ฿, ⠏⠍⠍⠆⠠⠼⠁⠚⠚⠀⠈⠃]
+ - [เลขไทย ๑๒๓๔๕๖๗๘๙, ⠋⠇⠅⠱⠾⠽⠀⠠⠼⠁⠃⠉⠙⠑⠋⠛⠓⠊]
+
+# backward test with 8 dots included in this file.
```

Comment:
What exactly does this comment mean?

Reply:
note: backward translation must use braille 8 dots only

Reply:
I changed the comment.